features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/MessagesReactionButton.kt

```diff
@@ -45,35 +46,50 @@ import io.element.android.libraries.theme.ElementTheme
 
 @Composable
 fun MessagesReactionButton(reaction: AggregatedReaction, modifier: Modifier = Modifier, onClick: () -> Unit) {
-    // First Surface is to render a border with the same background color as the background
-    Surface(
-        modifier = modifier.clickable(onClick = onClick::invoke),
+    val buttonColor = if(reaction.isHighlighted) {
         // TODO Should use compound.bgSubtlePrimary
-        color = ElementTheme.legacyColors.gray300,
-        border = BorderStroke(2.dp, MaterialTheme.colorScheme.background),
-        shape = RoundedCornerShape(corner = CornerSize(14.dp)),
+        ElementTheme.legacyColors.gray400
```

Comment:
I think now that compuond is available we can use the color `ElementTheme.colors.bgSubtlePrimary`. LegacyColors will be removed at some point.

Reply:
https://github.com/vector-im/element-x-android/pull/756/commits/8744cacb92f9eb3f332e84ca35873111beef167d